specs/pvs/corla.pvs

```diff
@@ -2622,8 +2625,8 @@ BEGIN
 %%% administrators using the RLA Tool, cause state changes. These
 %%% events are named according to their corresponding functional
 %%% definitions in earlier theories. E.g., the
-%%% \texttt{authenticate\_state\_administrator\_event} relates to the
-%%% \texttt{authenticate\_state\_administrator} function and its
+%%% \texttt{authenticate\_\-state\_\-administrator\_event} relates to
+%%% the \texttt{authenticate\_state\_administrator} function and its
```

Comment:
I already filed a ticket on this.  The authentication state and transition should be properly extracted from these ASMs.  See #213.